config/environments/production.rb

```diff
@@ -52,21 +53,15 @@
   # the I18n.default_locale when a translation cannot be found).
   config.i18n.fallbacks = true
 
-  # Send deprecation notices to registered listeners.
-  config.active_support.deprecation = :notify
-
-  # Log disallowed deprecations.
-  config.active_support.disallowed_deprecation = :log
-
-  # Tell Active Support which deprecation messages to disallow.
-  config.active_support.disallowed_deprecation_warnings = []
+  # Don't log any deprecations.
+  config.active_support.report_deprecations = false
```

Comment:
Yes, I just applied the new framework defaults. From what I can tell, the "notifications" are not consumed by any code, so I think that it doesn't makes sense to have them enabled. I think it's kind of waste of resources.
For disallowed deprecations the mode is `:log`, but then the list of disallowed deprecations is empty, so again it doesn't have any effect, I think.

So really we're not changing the behavior with this configuration change (i.e. we do not see deprecation warnings anywhere in production), just making Rails execute less code for the same behavior.

But if I'm wrong, please let me know! :)

Reply:
Why not enable `report_deprecations`?

Reply:
This is the new default for production in Rails 7, and it is the same behavior we've had until now - do nothing about deprecation warnings.

We can certainly consider whether we need this in future. But I think, given that the code base is very small, most if not all deprecation warnings should be caught in development and test stages.

Reply:
I think it's better to report anyway so we can always check the production logs. If we don't have deprecations, I assume, it will not have a performance impact.

Reply:
OK, I can open a separate PR for that :+1: